googleapiclient/model.py

```diff
@@ -273,13 +273,19 @@ def serialize(self, body_value):
         return json.dumps(body_value)
 
     def deserialize(self, content):
+        body = {}
         try:
             content = content.decode("utf-8")
         except AttributeError:
             pass
-        body = json.loads(content)
-        if self._data_wrapper and isinstance(body, dict) and "data" in body:
-            body = body["data"]
+        try:
+            body = json.loads(content)
+        except json.decoder.JSONDecodeError:
+            if isinstance(content, str):
+                body = content
```

Comment:
If the user passed in something other than `str`, we should just return it:
```suggestion
            body = content
```

Reply:
ok